client/blocks/disconnect-jetpack-dialog/docs/example.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import DisconnectJetpackDialog from '../';
+import Card from 'components/card';
+import Button from 'components/button';
+
+class DisconnectJetpackDialogExample extends Component {
+	constructor( props ) {
+		super( props );
+		this.toggleVisibilityFree = this.toggleVisibility.bind( this, 'free' );
+		this.toggleVisibilityPersonal = this.toggleVisibility.bind( this, 'personal' );
+		this.toggleVisibilityPremium = this.toggleVisibility.bind( this, 'premium' );
+		this.toggleVisibilityProfessional = this.toggleVisibility.bind( this, 'professional' );
+		this.toggleVisibilityBroken = this.toggleVisibility.bind( this, 'broken' );
+
+		this.state = {
+			isVisible: {
+				free: false,
+				personal: false,
+				premium: false,
+				professional: false,
+				broken: false
+			}
+		};
+	}
+
+	toggleVisibility( type ) {
+		this.state.isVisible[ type ] = ! this.state.isVisible[ type ];
+		this.setState( { isVisible: this.state.isVisible } );
+	}
+
+	render() {
+		return (
+			<Card>
+
+				<p><Button onClick={ this.toggleVisibilityFree } >Disconnect Free Jetpack </Button></p>
+				<DisconnectJetpackDialog
+					isVisible={ this.state.isVisible.free }
+					siteId={ this.props.primarySiteId }
+					onStay={ this.toggleVisibilityFree }
```

Comment:
I think it would be better to name this `onCancel`, even if the user facing wording is different.